cmd/skopeo/inspect.go

```diff
@@ -187,10 +201,31 @@ func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error)
 			logrus.Warnf("Registry disallows tag list retrieval; skipping")
 		}
 	}
-	out, err := json.MarshalIndent(outputData, "", "    ")
-	if err != nil {
+	if len(opts.format) == 0 {
+		out, err := json.MarshalIndent(outputData, "", "    ")
+		if err == nil {
+			fmt.Fprintf(stdout, "%s\n", string(out))
+		}
 		return err
 	}
-	fmt.Fprintf(stdout, "%s\n", string(out))
-	return nil
+	return outputFormat(outputData, opts.format)
+}
+
+func outputFormat(data interface{}, format string) error {
+	var out formats.Writer
+	if format == "json" || format == "" { // "" for backwards compat
```

Comment:
It’s a bit surprising to have a `json` option, when the output is JSON already by default, and for that option to mean “indent the output nicely”.

Not impossible, so if there is a strong precedent, I guess… still, a bit disappointing.

Reply:
I am removing, this is just stolen code from Buildah, which had this previously.